### PR TITLE
RHMAP-16030 - fhc services list --json doesn't work properly

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3652

Following the local test. 

screen shot 2017-06-02 at 10 21 10 am

### CODE REVIEW

**Comment by flashlee** [comment]
Command works good in terminal. But when I called it from code it returns me this array:
https://gist.github.com/flashlee/b1513a5189d46283c5f5b81ad9628bc5
This array contains "_table" with invalid format:
```
_table: Table {
```

**Comment by camilamacedo86** [comment]
@flashlee if you run the command into the fh-fhc and it worked perfect as you check either, by isolation the issue need be in the way that you are calling this into the another project.. In In my point if view, here is not the better place for we discuss it . Please, send to me the link for your code/project via any communication channel (email, irc, slack) that I will try to help you.

**Comment by camilamacedo86** [comment]
Hi, @jasonmadigan @kpiwko could you check this one? It is working fine into FHC, also @flashlee test either. His issue is in the call in another project.

**Comment by wtrocki** [comment]
@camilamacedo86 Just to provide context. Technically fh-fhc can be also used as node.js library. 
When making changes we usually test it using command line, but this particular issue may also affect library. 

In actual PR I think that PR can be improved. fh-fhc has settings to format results automatically depending on customer flag (globally):
https://github.com/feedhenry/fh-fhc/blob/master/lib/utils/result-handler.js#L22-L36

This code is far from ideal and also begging for small refactoring, but... I believe that hardcoding this to each command is not the way to go. Feel free to research new yargs - it introduces totally different format, support for folders and global flags, which may be perfect example for handling console output. 

TL;DR - adding output flag to single command is no go. We need to reuse result-handler or global handlers from yargs.

**Comment by camilamacedo86** [comment]
@wtrocki as you sent here this is a global configuration[1] and the param in the code is just a output in the help to show that is possible. The fix here is don't use the customCmd for this global code be used. 

In my point if view, If we need change the way that the app works related of this is not in this PR. We need create a JIRA for do it and follow the flow. Could you create this JIRA and put or your thoughts and how do you think we should improve it?

 [1] - https://github.com/feedhenry/fh-fhc/blob/master/lib/utils/result-handler.js#L22-L36

File: lib/cmd/fh3/services/list.js [hunk 1]
```diff
@@ -16,13 +16,10 @@ module.exports = {
   'describe' : {
     'json' : i18n._('Output into json format')
```
Comment (wtrocki) [inline lib/cmd/fh3/services/list.js]: I assumed that this was added in PR. 
What I meant is that we can set this flag to be available globally.

**Comment by wtrocki** [comment]
@camilamacedo86 - I think we may also need to cover use cases when fh-fhc is used as library.

File: lib/cmd/fh3/services/list.js (hunk 1 above)
Comment (camilamacedo86) [inline lib/cmd/fh3/services/list.js]: 👍 I create the JIRA for this: https://issues.jboss.org/browse/FH-3650
Please, feel free to contribute with or thoughts there.

**Comment by camilamacedo86** [comment]
Hi, @wtrocki tks for you review and contribution here 👍 

About your last comment: "I think we may also need to cover use cases when fh-fhc is used as library." Please, could you open a JIRA with your ideia? it is not clear for me because I can't see how we can do this inside of fhc. Also, I believe that @flashlee is coding exactly it, tests for this project and he is with some difficult to carry out this implementation, which I see that we need first understand his difficulties for after check if is required or not do some change in the project. If yes, we need to created a JIRA and flow the process because probably it will be a global change.

**Comment by kpiwko** [comment]
Created a follow up issue here on Serhii's behalf: https://issues.jboss.org/projects/FH/issues/FH-3655
